tests/integ/modin/series/test_dt_accessor.py

```diff
@@ -73,7 +74,7 @@ def set_week_start(request):
     "datetime_index_value",
     [
         ["2014-04-04 23:56", "2014-07-18 21:24", "2015-11-22 22:14"],
-        ["04/04/2014", "07/18/2013", "11/22/2015"],
```

Comment:
Why change these years?

Reply:
To include a leap year in them.

Reply:
This method is checking `dt.date`, though. Would the result of `dt.date` depend on whether the date is in a leap year?

Reply:
I could have added one extra date in a leap year instead. Do you prefer that?

Reply:
I don't see how that gives coverage for `is_leap_year`, either. Can we revert this line?

Reply:
Oh, you're right. Will revert. I must have thought that this was run against all properties. Thanks for the catch!

Reply:
Done.